packages/discovery-provider/src/tasks/entity_manager/entities/playlist.py

```diff
@@ -287,6 +287,55 @@ def update_album_price_history(
             session.add(new_record)
 
 
+def populate_playlist_record_metadata(
```

Comment:
it seems like `process_playlist_data_event` is doing a similar thing, can this be moved there?